gcp_variant_transforms/testing/integration/small_tests/merge_option_copy_filter_to_calls.json

```diff
@@ -0,0 +1,58 @@
+{
+  "test_name": "merge-option-copy-filter-to-calls",
+  "table_name": "merge_option_copy_filter_to_calls",
+  "input_pattern": "gs://gcp-variant-transforms-testfiles/small_tests/merge/*.vcf",
+  "variant_merge_strategy": "MOVE_TO_CALLS",
+  "copy_filter_to_calls": true,
+  "runner": "DataflowRunner",
+  "assertion_configs": [
+    {
+      "query": ["NUM_ROWS_QUERY"],
+      "expected_result": {"num_rows": 4}
+    },
+    {
+      "query": ["SUM_START_QUERY"],
+      "expected_result": {"sum_start": 1283553}
+    },
+    {
+      "query": ["SUM_END_QUERY"],
+      "expected_result": {"sum_end": 1283560}
+    },
+    {
+      "query": [
+        "SELECT COUNT(0) AS num_rows ",
+        "FROM {TABLE_NAME} AS t, t.call as call ",
+        "WHERE start_position = 14369 AND call.name ='NA00001' ",
```

Comment:
Consider verifying position 1234567 as well since you expect different number of calls.

Reply:
I am not trying to test the number of calls. Instead, what I try to test is that we have a column call.filter in the BQ table when `copy_filter_to_calls` is set to true, and the value is copied from the original vcf file. Am I in the wrong direction?

Reply:
Thanks for clarifying, I misunderstood what you are trying to do; and yes, both for this comment and the next one, I was thinking about testing `MOVE_TO_CALLS` merge strategy and checking that calls are merged. Up to you whether you want to test call names in that test or not, I am okay either way.

Reply:
Thanks. I have added a test case for `merge_option_move _to_calls` to check that the calls are merged for one specific position. Meanwhile, I added a similar test case in `merge_option_none` to validate that the calls are not merged.